ofl/notoserifkhitansmallscript/METADATA.pb

```diff
@@ -0,0 +1,32 @@
+name: "Noto Serif Khitan Small Script"
+designer: "Google"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2023-01-11"
+fonts {
+  name: "Noto Serif Khitan Small Script"
+  style: "normal"
+  weight: 400
+  filename: "NotoSerifKhitanSmallScript-Regular.ttf"
+  post_script_name: "NotoSerifKhitanSmallScript-Regular"
+  full_name: "Noto Serif Khitan Small Script Regular"
+  copyright: "Copyright 2022 Google LLC. All Rights Reserved."
+}
+subsets: "khitan-small-script"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/khitan-small-script.git"
+  archive_url: "https://github.com/notofonts/khitan-small-script/releases/download/NotoSerifKhitanSmallScript-v1.000/NotoSerifKhitanSmallScript-v1.000.zip"
+}
+is_noto: true
+sample_text {
+  masthead_full: "𘭔𖿤𘭢 𘲽𘱸"
+  masthead_partial: "𘰷𘭴"
+  styles: "𘰷𘭴 𘰝𘲥𘲦 𘲲𘯴𘱆 𘳍𘰄 𘰴𘳍𘰄 𘬜𘭞𘰷 𘬜𘭞𘰷 𘰷𘭞𘭟 𘰷𘭞𘭟 𘱓𘮯𘮡"
```

Comment:
Fonts eng requested that I edit the length of this string to address the very tall appearance of this family on the front end.  Will open a new PR with these changes.